lib/test/clojure_lsp/feature/add_missing_libspec_test.clj

```diff
@@ -211,6 +211,7 @@
     (testing "Do not add from wrong language"
```

Comment:
This test was passing not because of the mismatch between languages as its name suggests, but because `foo.s` wasn't defined and because undefined namespaces aren't suggested. Adding the definition of `foo.s` in a cljs file made it fail and the rest of this patch is the fix.